repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -470,27 +470,27 @@ public EntityMutationResponse mapAttributesAndClassifications(EntityMutationCont
                     throw baseException;
                 }
             }
-        } else {
+        }
 
-            if (CollectionUtils.isNotEmpty(appendEntities)) {
-                for (AtlasEntity entity : appendEntities) {
-                    String guid = entity.getGuid();
-                    AtlasVertex vertex = context.getVertex(guid);
-                    AtlasEntityType entityType = context.getType(guid);
-                    mapAppendRemoveRelationshipAttributes(entity, entityType, vertex, UPDATE, context, true, false);
-                }
+        if (CollectionUtils.isNotEmpty(appendEntities)) {
```

Comment:
Won't this create multiple updates or multiple entity audits for the same entity? Can you verify the same? Basically if an update for an entity has a primitive attribute update and an append relationship, what happens to audits?